templates/convergedstoragecluster.go

```diff
@@ -17,8 +17,8 @@ limitations under the License.
 package templates
```

Comment:
This will not be required. Why keep this?

Reply:
Removed in the latest commit